config/systemConfig/java.yaml

```diff
@@ -7,3 +7,6 @@ systemConfig:
   
   - key: apiSinks
     value: (?i)(?:url|client|openConnection|request|execute|newCall|load|host|access|fetch|get|getInputStream|getApod|getForObject|getForEntity|list|set|put|post|proceed|trace|patch|Path|send|sendAsync|remove|delete|write|read|assignment|provider|exchange|postForEntity)
+
+  - key: apiIdentifier
+    value: (?i).*((hook|base|authorize|provider|endpoint|installation)(s){0,1}_url|(slack|web)_hook|(rest|api)_endpoint).*
```

Comment:
Any reason to have all these regex not match CamelCase

Reply:
Let me update the regex to have `_` as optional